pkg/neorpc/result/version.go

```diff
@@ -64,6 +65,11 @@ type (
 		ValidatorsHistory map[uint32]uint32
 	}
 
+	Application struct {
+		SaveInvocations     bool `json:"saveinvocations"`
+		KeepOnlyLatestState bool `json:"keeponlylateststate"`
```

Comment:
Use `omitempty` json tag to follow the existing behaviour of `getversion` extensions:

> These fields are only returned when corresponding settings are enabled in the server's protocol configuration.

E.g.: https://github.com/nspcc-dev/neo-go/blob/2971b8a1c7dfc8cb895f9a60e8f1f36c60020948/pkg/neorpc/result/version.go#L83